crates/db_schema/src/test_utils.rs

```diff
@@ -0,0 +1,98 @@
+#![allow(clippy::unwrap_used)]
```

Comment:
What does this have to do with this PR? Remove this.

Reply:
Clippy complains otherwise. This is test code but I think the `unwrap` uses could be just changed to `expect`